packages/terra-site/webpack.config.js

```diff
@@ -48,11 +48,9 @@ module.exports = {
               ];
             },
           },
-        }, {
+        },
+        {
           loader: 'sass-loader',
-          options: {
-            data: `@import "${path.resolve(path.join(__dirname, 'node_modules/terra-legacy-theme/lib/LegacyTheme.scss'))}"; $terra-bidi: true;`,
```

Comment:
https://github.com/cerner/terra-core/pull/450/files#diff-a0157ec3f4609e3df477bbd4bf487b52L4

nm, I see it is hardcoded where needed